### ISSUE TITLE
Building documentation raises AttributeError: 'NoneType' object has no attribute 'extend'

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#1157]
Some of our sphinx projects do not build on my Mac with the following exception:

```
Exception occurred:
  File "/usr/local/lib/python2.7/site-packages/sphinx/environment.py", line 1046, in get_toctree_for
    result.extend(toctree.children)
AttributeError: 'NoneType' object has no attribute 'extend'
```

Attached is also the full log for this error.

Strangely, some other project (probably a bit simpler) work without problems. However, all of the projects can be built on other computers. So there must be a specific issue with my configuration.

This is my local confiuguration:

```
languitar@ip-129-70-134-168:~/work/src/rst/rst-manual/build$ pip show sphinx

---
Name: Sphinx
Version: 1.2b1
Location: /usr/local/lib/python2.7/site-packages
Requires: Pygments, Jinja2, docutils
languitar@ip-129-70-134-168:~/work/src/rst/rst-manual/build$ pip show Pygments

---
Name: Pygments
Version: 1.6
Location: /usr/local/lib/python2.7/site-packages
Requires:
languitar@ip-129-70-134-168:~/work/src/rst/rst-manual/build$ pip show Jinja2

---
Name: Jinja2
Version: 2.6
Location: /usr/local/lib/python2.7/site-packages
Requires:
languitar@ip-129-70-134-168:~/work/src/rst/rst-manual/build$ pip show docutils

---
Name: docutils
Version: 0.10
Location: /usr/local/lib/python2.7/site-packages
Requires:
```

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/1157
- Originally reported by: [languitar](https://bitbucket.org/languitar)
- Originally created at: 2013-04-30T15:00:54.102

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2013-04-30 13:57:36+00:00_

[languitar](https://bitbucket.org/languitar), thanks for reporting. It's a bug.

The combination of 'globaltoc.html' and hidden toctree cause this issue.

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2013-04-30 14:36:47+00:00_

Fix: Combination of 'globaltoc.html' and hidden toctree cause exception. Closes #1157

→ <<cset 59d655c435e7ca1a5e48cbbf9809d2553cc71d5d>>

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2013-05-01 21:49:54+00:00_

Issue #1141 was marked as a duplicate of this issue.

**Comment by shimizukawa** [comment]
_From [languitar](https://bitbucket.org/languitar) on 2013-05-15 15:35:41+00:00_

In which version of sphinx will this be available?

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2013-05-17 06:56:52+00:00_

[languitar](https://bitbucket.org/languitar) Sphinx-1.2 will contains this fix.

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2013-09-17 13:45:14+00:00_

Issue #1263 was marked as a duplicate of this issue.
